Challenges/Arrays/ArrayShift.cs

```diff
@@ -4,8 +4,19 @@ namespace Challenges.Arrays
 {
     public class ArrayShift
     {
-        public ArrayShift()
+        public ArrayShift
```

Comment:
This should be "public int[] InsertShiftArray() {...}.  How you have it right now is setup as a constructor.  It cannot be named the same as your class.